apps/epic-react/src/pages/workshops/[module]/[lesson]/index.tsx

```diff
@@ -6,21 +6,15 @@ import {VideoResourceProvider} from '@skillrecordings/skill-lesson/hooks/use-vid
 import {LessonProvider} from '@skillrecordings/skill-lesson/hooks/use-lesson'
 import {ModuleProgressProvider} from '@skillrecordings/skill-lesson/video/module-progress'
 import {getSection} from '@/lib/sections'
-import serializeMDX from '@skillrecordings/skill-lesson/markdown/serialize-mdx'
 import {getAllWorkshops, getWorkshop} from '@/lib/workshops'
-import {getAllBonuses, getBonus} from '@/lib/bonuses'
 import {serialize} from 'next-mdx-remote/serialize'
 
 export const getStaticProps: GetStaticProps = async (context) => {
   const {params} = context
   const lessonSlug = params?.lesson as string
   const sectionSlug = params?.section as string
   const moduleSlug = params?.module as string
-  const isBonusModule = moduleSlug === 'epic-react-expert-interviews'
-
```

Comment:
@zacjones93 Both bonuses and workshops lessons are located by `/modules/[module-slug]/[lesson-slug]` address at original EpicReact. It is why I get both types of resources in the same template (Lesson template). Only the thing that needs to be done is replace `/workshops/` with `/modules/` to be consistent with original ER. So I don't think it is good idea to have separate dynamic route `bonuses/[bonus]/[interview]` because it's unclear how to keep the same url as workshops have. What do you think @jbranchaud ?

Reply:
Ah ok, that makes sense. I misunderstood the requirement there

Reply:
@DrShpongle yeah, we want parity on the URLs for sure. We could always create framework-level redirects in the `next.config.js`, but probably easier to match URLs now with the file structure.